shared/actions/profile/proofs.js

```diff
@@ -145,9 +146,32 @@ function* addProof(state, action) {
     }
   })
 
+  const watchForNavUp = yield Saga._fork(function*() {
+    // TODO remove this when nav1 is removed
```

Comment:
We're probably not going to notice this TODO later, but if you put a mention of flags.useNewRouter in here that'd show up when we grep to remove the feature flag.. even if that mention's inside a comment, I guess!

Reply:
Good point, thanks, I'll add it

Reply:
@cjb also swapped out the call to `checkProof` with a call to `Identify3` when `startProof` finishes